### DISCUSSION TITLE
Facet Plots

### DISCUSSION THREAD

**Discussion by usrbinr** [bprusinowski/plotteus#11] [General]
Hi!

I am new to observable.js but I got really excited by the blog post below on how I can use Quarto, R and your package to create interactive plots. 

It's actually perfect for the reveal.js slides that I make at work. 

While I think I understand the arguments in how to use your package, my question is there a way to do facet a plot (along a group) or are we limited to the arguments that you have described in your post?

https://github.com/hrbrmstr/hello-plotteus

**Comment by bprusinowski** [comment]
Hi @alejandrohagan! I am glad to hear that you like the library :)

While it's not a native feature of Plotteus to create sub-plots (although it's on my todo list!), you can achieve the same result by creating multiple charts and arranging them in any way you like. This requires some work to prepare the data for each plot (and e.g. make sure the scales share the same domain), but should hopefully be sufficient for common needs. I've added and example of how to do this for a bar chart [here](https://github.com/bprusinowski/plotteus/blob/main/examples/facets.html).

Hope it helps!

**Reply by usrbinr** [reply]
Thank you!
